src/main/java/com/aws/greengrass/deployment/converter/DeploymentDocumentConverter.java

```diff
@@ -330,6 +331,14 @@ private static FailureHandlingPolicy convertFailureHandlingPolicy(
         return FailureHandlingPolicy.valueOf(failureHandlingPolicy.name());
     }
 
+    private static FailureHandlingPolicy convertFailureHandlingPolicyFromSDK(
+            software.amazon.awssdk.aws.greengrass.model.FailureHandlingPolicy failureHandlingPolicy) {
```

Comment:
Why can't we directly import the library at the very top?

Reply:
Because there is another `FailureHandlingPolicy` exactly the same wording imported from the top [import com.aws.greengrass.deployment.model.FailureHandlingPolicy](https://github.com/aws-greengrass/aws-greengrass-nucleus/blob/9aa7c1242a786f0cc4624a473d93da11067c28d1/src/test/java/com/aws/greengrass/deployment/converter/DeploymentDocumentConverterTest.java#L17) (Should I change the model name for `software.amazon.awssdk.aws.greengrass.model.FailureHandlingPolicy` to avoid this type of reference?)

Reply:
Aha, AFAIK there is no way to reference 2 different paths with same name. I am ok we keep this.